### PR TITLE
Added TSR Hub to serverlist

### PR DESCRIPTION
**Pull request by GaviTSRA**
Added the tsr hub to the server list

### CODE REVIEW

File: servers_v6.json [hunk 1]
```diff
@@ -93,6 +93,11 @@
   },
   {
     "name": "thedimas",
-    "address": ["95.217.226.152:26160"]
+
+    "address": ["95.217.226.152:26160"]
```
Comment (Quezler) [inline servers_v6.json:97]:
```suggestion
    "address": ["95.217.226.152:26160"] 
```

File: servers_v6.json [hunk 2]
```diff
@@ -93,6 +93,10 @@
   },
   {
     "name": "thedimas",
-    "address": ["95.217.226.152:26160"]
+    "address": ["95.217.226.152:26160"]
```
Comment (Quezler) [inline servers_v6.json:96]:
```suggestion
    "address": ["95.217.226.152:26160"]
```

**Comment by Anuken** [comment]
Server down:
image

**Comment by GaviTSRA** [comment]
> Server down:
> image

Yes, my host currently has problem with their service (I can't so anything about that) should be back up in some time (max 1 - 2 days, I think a few hours)

**Comment by GaviTSRA** [comment]
> Server down:
> image

New hub, one which isn't offline. You can have a look at it now

**Pull request merged**